ui/packages/consul-ui/app/templates/unavailable.hbs

```diff
@@ -0,0 +1,108 @@
+              fill='#E03875'
+            />
+            <path
+              d='M33.0432 18.5008C32.7245 18.5008 32.4129 18.4079 32.1479 18.2338C31.8828 18.0597 31.6763 17.8122 31.5543 17.5227C31.4323 17.2332 31.4004 16.9146 31.4626 16.6073C31.5248 16.3 31.6783 16.0177 31.9037 15.7961C32.129 15.5745 32.4162 15.4236 32.7288 15.3625C33.0414 15.3013 33.3655 15.3327 33.6599 15.4526C33.9544 15.5726 34.2061 15.7756 34.3832 16.0362C34.5603 16.2967 34.6548 16.603 34.6548 16.9164C34.6548 17.3366 34.485 17.7396 34.1828 18.0367C33.8805 18.3339 33.4706 18.5008 33.0432 18.5008Z'
+              fill='#E03875'
+            />
+            <path
+              d='M36.0803 12.9856C35.7616 12.9856 35.45 12.8927 35.185 12.7186C34.92 12.5445 34.7134 12.2971 34.5914 12.0076C34.4694 11.7181 34.4375 11.3995 34.4997 11.0922C34.5619 10.7848 34.7154 10.5025 34.9408 10.2809C35.1662 10.0594 35.4533 9.90847 35.7659 9.84734C36.0785 9.7862 36.4026 9.81758 36.697 9.9375C36.9915 10.0574 37.2432 10.2605 37.4203 10.521C37.5974 10.7816 37.6919 11.0879 37.6919 11.4013C37.6919 11.8215 37.5221 12.2245 37.2199 12.5216C36.9176 12.8187 36.5077 12.9856 36.0803 12.9856Z'
+              fill='#E03875'
+            />
+          </g>
+          <defs>
+            <clipPath id='clip0_2336_115454'>
+              <rect width='150' height='39' fill='white' />
+            </clipPath>
+          </defs>
+        </svg>
+
+        <Hds::ApplicationState as |A|>
+          <A.Header @title='User Interface Unavailable' data-test-unavailable-header />
+          <A.Body
+            @text='The Consul v2 catalog API (beta) is enabled, and is not yet compatible with the Consul UI.'
+            data-test-unavailable-body
+          />
+          <A.Footer @hasDivider={{true}} as |F|>
+            <F.Link::Standalone
+              @icon='docs-link'
+              @iconPosition='trailing'
+              @isHrefExternal={{true}}
+              @text='Learn more'
+              @href='https://developer.hashicorp.com/consul/docs/architecture/catalog/v2#constraints-and-limitations'
+            />
+            <F.Link::Standalone
+              @icon='external-link'
+              @iconPosition='trailing'
+              @isHrefExternal={{true}}
+              @text='Provide feedback'
+              @href='https://hashicorp.sjc1.qualtrics.com/jfe/form/SV_cHDaObigIBbQneC'
+            />
+          </A.Footer>
+        </Hds::ApplicationState>
+      </div>
+    </div>
+  {{else}}
+    {{did-insert (route-action 'replaceWith' 'index')}}
```

Comment:
Don't we also do this in the `route`?

Reply:
You are very correct, I forgot to remove this once I added the route. I will do that on Monday. Thanks!